gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/task/LaunchDagTask.java

```diff
@@ -36,4 +42,25 @@ public LaunchDagTask(DagActionStore.DagAction dagAction, LeaseAttemptStatus.Leas
   public <T> T host(DagTaskVisitor<T> visitor) {
     return visitor.meet(this);
   }
+
+  @Override
+  public final boolean conclude() {
+    try {
+      // Delete adhoc flowSpecs from catalog if the dag was concluded properly
```

Comment:
You probably want to rename this comment to be "remove adhoc flowspecs after the adhoc job is launched", to be more accurate to what we want out of the behavior